.travis.yml

```diff
@@ -26,7 +26,7 @@ before_script:
 install:
   - travis_retry composer -n install
 
-script: ./vendor/bin/phpunit --configuration tests/travis/$DB.travis.xml
+script: SYMFONY_DEPRECATIONS_HELPER=weak_vendors ./vendor/bin/phpunit --configuration tests/travis/$DB.travis.xml
```

Comment:
Not using the `simple-phpunit` binary because of a crash I don't understand (seems to have to do with autoloading).
The "weak_vendors" will make the tests suite fail if I forget to remove any call to setOption with an unknown option, unless it's in a test marked with the "@legacy" annotation. The test suite won't fail if a vendor suddenly adds a deprecation (that's why it's called weak vendors, it's lenient with deprecations from vendors)

Reply:
I pushed something new. Turns out you don't have to use `simple-phpunit` to test deprecations properly.